modules/dnn/test/test_layers.cpp

```diff
@@ -2742,7 +2742,7 @@ INSTANTIATE_TEST_CASE_P(TestLayerFusion, ConvolutionActivationEltwiseFusion, Com
 
 TEST(Layer_LSTM, repeatedInference)
 {
-    std::string onnx_file_path = findDataFile("dnn/onnx/models/onnxscript_lstm.onnx", false);
+    std::string onnx_file_path = findDataFile("dnn/onnx/models/onnxscript_lstm.onnx", true);
```

Comment:
`true` value is a default, so it could be removed.

Reproduce with `OPENCV_TEST_CHECK_OPTIONAL_DATA=1` environment variable.